### ISSUE TITLE
Should this just be for internal posts? 

### ISSUE THREAD

**Issue by ldabiralai** [sky-uk/developers-sky-com#146]
I don't think we should try and cater for both internal and external users. The main purpose should be to attract new talent to Sky.

We have things like Slack for internal only communication.

It'll also mean we can keep the repo public in case people are interested in the technology behind it, and it's easier and more beneficial for us to contribute to it.

**Comment by regularfry** [comment]
I disagree.  The main purpose should be to provide information *for* developers at sky; publishing externally enticing content will fall naturally out of that.  We want a single location for longer-form technical pieces for an internal audience.  Slack isn't suited to that.

I'm also working on a new top-level area on developers.sky.com which would be explicitly internal-only, but relies on developers.sky.com being a central port of call for long-lived developer-focused info.

We *could* fork, and maintain `internal-` and `external-` forks long-term, copying articles between the two, if doing the public/private separation is too technically unwieldy.

**Comment by andrewmerrell** [comment]
At our last catch up we agreed our core focus this month is to drive the site to launch, we want to start showcasing the excellent work of our development community and use it as a platform for recruitment.

I believe this can only happen if we remove the internal/external bounds and I'm inclined to agree with @ldabiralai for the short term. The current vision we've been working towards is for recruitment, but I agree it should be much more than that long term.

@regularfry your suggestion around forks seems appealing at the moment as it is near impossible to achieve both internal and external on the github platform without everything being automatically external.

Sounds like you have some ideas around the long term strategy, would be great if you could present  them at the next developers-sky-com catch up?

**Comment by craggy2593** [comment]
@andrewmerrell Talks of separate repos for internal and external was something we considered when developing, and particularly when we found our Jekyll plugin to distinguish between the two wouldn't work on GitHub. When we spoke to Hubert about it, he suggested having an internal site being hosted on something like Nimbus, and the external site hosted on the gh-pages branch, but ultimately all still maintained from the single repo. It might be worth speaking to him again, as he mentioned his blog works in this exact way.

If we were to also consider moving across to Polaris, with it being hosted elsewhere from GitHub, and Jekyll was still to be used, then our plugin for IP filtering could be used again. It was a far more robust solution than the JS solution we ended up implementing.

**Comment by behaghel** [comment]
I am with @regularfry. It is my humble opinion that it's really good to make it internal as well as external. The main reason being that we want to showcase our culture, something we relate to. I believe real software engineers do post regularly but not every Sky software engineer will spin up a blog like [Le Carnet](http://le-carnet.sky.com). So this gives every one a place to post. 

At the minute, I think we are doing ok in terms of editorial line and our collective writing skill is decent. But not where I'd like it to be :wink: By encouraging each other to post more and practice this art, by forming a community around this repo to review and sharpen our relevance, we should maximise the number of submissions, increase the quality of content and then distillate the very best of it to become the external face of our developer culture.

Technically I see many ways to do it and for an MVP, the current approach is good enough IMHO. I agree it's not perfect @andrewmerrell but isn't it good enough?

A single repo is important as it will simplify things. As such I am not inclined to work with forks. I think all posts should be internal first and would recommend we stop from suggesting to create external posts in the README.

**Comment by andrewmerrell** [comment]
Agreed I also prefer not to work from forks, I would choose a branch strategy over a fork one if this is the route we took, master considered internal could be deployed on Nimbus as mentioned by @craggy2593 and gh-pages considered the external.

Ultimately I'd rather not worry, my main concern is more around discoverability of internal posts is trivial, and I hate us to fall foul if they don't meet the current external guidelines (I haven't read them all) - But maybe I worry too much and need to take the plunge?

**Comment by behaghel** [comment]
@craggy2593 I guess we still have Google Analytics on the blog right? 

If so @andrewmerrell, I would take the plunge and monitor those internal posts. If they get traction from the outside world, then I would take action. But frankly, I have read most of them and I can't think of it becoming an issue of any kind.

**Comment by andrewmerrell** [comment]
Good idea @behaghel, If someone could provide me access that would be much appreciated!

**Comment by craggy2593** [comment]
@behaghel Yeah GA is still on the blog, @andrewmerrell Can't add your Sky email address, do you have a Google account for it?

**Comment by behaghel** [comment]
Can you add mine as well please Craig? behaghel@gmail.com

On 01/02/2016 12:25, "Craig" <notifications@github.com<mailto:notifications@github.com>> wrote:


@behaghel<https://github.com/behaghel> Yeah GA is still on the blog, @andrewmerrell<https://github.com/andrewmerrell> Can't add your Sky email address, do you have a Google account for it?

—
Reply to this email directly or view it on GitHub<https://github.com/sky-uk/developers-sky-com/issues/146#issuecomment-177949044>.

Information in this email including any attachments may be privileged, confidential and is intended exclusively for the addressee. The views expressed may not be official policy, but the personal views of the originator. If you have received it in error, please notify the sender by return e-mail and delete it from your system. You should not reproduce, distribute, store, retransmit, use or disclose its contents to anyone. Please note we reserve the right to monitor all e-mail communication through our internal and external networks. SKY and the SKY marks are trademarks of Sky plc and Sky International AG and are used under licence. Sky UK Limited (Registration No. 2906991), Sky-In-Home Service Limited (Registration No. 2067075) and Sky Subscribers Services Limited (Registration No. 2340150) are direct or indirect subsidiaries of Sky plc (Registration No. 2247735). All of the companies mentioned in this paragraph are incorporated in England and Wales and share the same registered office at Grant Way, Isleworth, Middlesex TW7 5QD.

**Comment by craggy2593** [comment]
@behaghel You should already have an account, it says you're registered
